test/IRCTest.h

```diff
@@ -188,6 +198,7 @@ class TestModule : public CModule {
     std::vector<CClient*> vClients;
     std::vector<CChan*> vChannels;
     EModRet eAction = CONTINUE;
+    bool bSendHooks = false;
```

Comment:
I did this so that, when testing other hooks, the `OnSendToClient/IRCMessage` hooks don't fire. Before I put it in here, it was necessary to modify other tests to indicate that `OnSendToClient/IRCMessage` would be included in the list of hooks that fired. Additionally, it simplified handling of `HALT` for the tests.